src/overlays/actors/ovl_Dm_Zl/z_dm_zl.h

```diff
@@ -8,12 +8,25 @@ struct DmZl;
 typedef void (*DmZlActionFunc)(struct DmZl*, GlobalContext*);
 
 typedef struct DmZl {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x11C];
-    /* 0x0260 */ DmZlActionFunc actionFunc;
-    /* 0x0264 */ char unk_264[0x70];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ UNK_TYPE1 pad188[0xD8]; // unused by DmZl
+    /* 0x260 */ DmZlActionFunc actionFunc;
+    /* 0x264 */ UNK_TYPE1 pad264[0x4C]; // unused by DmZl
+    /* 0x2B0 */ s16 animationIndex;
+    /* 0x2B2 */ u8 eyeTextureIndexLeft;
+    /* 0x2B3 */ u8 eyeTextureIndexRight;
+    /* 0x2B4 */ u8 mouthTextureIndex;
+    /* 0x2B5 */ u8 nextEyeState;   // used to control eye state, but not set by our actor, outside of actor?
+    /* 0x2B6 */ u8 nextMouthState; // used to control mouth state, but not set by our actor, outside of actor?
+    /* 0x2B8 */ s16 blinkTimer;
+    /* 0x2BA */ s16 unused2BA; // init to 0, unread/unchanged by DmZl after
+    /* 0x2BC */ UNK_TYPE1 pad2BC[0x18]; // unused by DmZl
 } DmZl; // size = 0x2D4
 
 extern const ActorInit Dm_Zl_InitVars;
 
+// vanilla params are 0x0000, never checked or used
+#define DMZL_TYPE_SOT_CUTCSENE 0
```

Comment:
Since the params are never used, it's probably not worth having this... does anyone feel strongly either way?

Reply:
Don't feel strongly, though I would probably lean towards removing it.

Reply:
I was leaning the other way, I think for modding its important to know what parameters are actually used in the vanilla game to know what changes you made to params input would be affected if the vanilla actor was encountered.

But I can remove it.